ttnn/cpp/ttnn/operations/moreh/moreh_getitem/device/moreh_getitem_tilized_factory.cpp

```diff
@@ -82,49 +82,46 @@ operation::ProgramWithCallbacks moreh_getitem_tilized(
 
         for (uint32_t i = 0; i < index_tensors.size(); i++) {
             auto dim = index_dims[i] + input_dim_offset;
-            auto index = index_tensors.at(i);
+            auto index = index_tensors[i];
 
             index_info[dim].is_defined = true;
             index_info[dim].address = index.buffer()->address();
-            index_info[dim].is_dram = is_dram(index);
+            index_info[dim].is_dram = tt::operations::primary::is_dram(index);
             index_info[dim].unit_size = index.element_size();
         }
 
-        uint32_t index_size = index_tensors.at(0).get_legacy_shape().without_padding()[-1];
+        uint32_t index_size = index_tensors[0].get_shape().value.without_padding()[-1];
 
         uint32_t input_unit_size = input.element_size();
         uint32_t output_unit_size = output.element_size();
 
-        // split work
-        auto input_5d_shape_without_padding = input_5d_shape.without_padding();
-        auto output_5d_shape_without_padding = output_5d_shape.without_padding();
         uint32_t alignment_size = 32;
         uint32_t num_elements_per_alignment = alignment_size / output_unit_size;
         uint32_t num_units =
-            output_5d_shape_without_padding[0] * output_5d_shape_without_padding[1] * output_5d_shape_without_padding[2] *
-            output_5d_shape_without_padding[3] * ((output_5d_shape_without_padding[4] + num_elements_per_alignment - 1) / num_elements_per_alignment);
-        log_debug(LogTest, "num_units {}", num_units);
+            output_5d_shape_without_padding[0] * output_5d_shape_without_padding[1] *
+            output_5d_shape_without_padding[2] * output_5d_shape_without_padding[3] *
+            ((output_5d_shape_without_padding[4] + num_elements_per_alignment - 1) / num_elements_per_alignment);
 
         uint32_t core_w = core_range.end_coord.x - core_range.start_coord.x + 1;
         uint32_t core_h = core_range.end_coord.y - core_range.start_coord.y + 1;
 
         auto
             [num_cores, all_cores, core_group_1, core_group_2, num_units_per_core_group_1, num_units_per_core_group_2] =
-                split_work_to_cores(core_range, num_units);
+                tt::operations::primary::split_work_to_cores(core_range, num_units);
 
         Program program = Program();
 
         // create circular buffers
-        auto src_cb_data_format = tt_metal::datatype_to_dataformat_converter(input.get_dtype());
-        auto index_cb_data_format = tt_metal::datatype_to_dataformat_converter(index_tensors.at(0).get_dtype());
-        auto output_cb_data_format = tt_metal::datatype_to_dataformat_converter(output.get_dtype());
+        auto src_cb_data_format = tt::tt_metal::datatype_to_dataformat_converter(input.get_dtype());
+        auto index_cb_data_format = tt::tt_metal::datatype_to_dataformat_converter(index_tensors[0].get_dtype());
+        auto output_cb_data_format = tt::tt_metal::datatype_to_dataformat_converter(output.get_dtype());
```

Comment:
```suggestion
        using namespace tt;
        using namespace tt::tt_metal;
    
        auto src_cb_data_format = datatype_to_dataformat_converter(input.get_dtype());
        auto index_cb_data_format = datatype_to_dataformat_converter(index_tensors[0].get_dtype());
        auto output_cb_data_format = datatype_to_dataformat_converter(output.get_dtype());
```
https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/examples/example/device/single_core_program_factory.cpp#L13

adding `using namespace tt;` could reduce the amount of code.